go/test/endtoend/cluster/cluster_util.go

```diff
@@ -94,7 +94,7 @@ func GetPrimaryPosition(t *testing.T, vttablet Vttablet, hostname string) (strin
 // VerifyRowsInTabletForTable verifies the total number of rows in a table.
 // This is used to check that replication has caught up with the changes on primary.
 func VerifyRowsInTabletForTable(t *testing.T, vttablet *Vttablet, ksName string, expectedRows int, tableName string) {
-	timeout := time.Now().Add(10 * time.Second)
+	timeout := time.Now().Add(1 * time.Minute)
```

Comment:
This timeout has been increased for VTOrc. With the defaults, it takes VTOrc 15 seconds to discover the tablets and then it can start repairing them. This wait was for 10 seconds which turned out to be too small. I have improved VTOrc performance ☝️, but it doesn't hurt to increase the timeout.